tests/TestCase.php

```diff
@@ -70,10 +77,12 @@ protected function setUp(): void
         $this->aliases = $this->container->get(Aliases::class);
         $this->assetManager = $this->container->get(AssetManager::class);
         $this->eventDispatcher = $this->container->get(EventDispatcherInterface::class);
+        $this->listenerProvider = $this->container->get(ListenerProviderInterface::class);
```

Comment:
Seems `listenerProvider` isn't declared as class property.

Reply:
https://github.com/terabytesoftw/view/blob/d2334acb642467f4ee4944c83ebe0d1829ef3b98/config/tests.php#L56

Reply:
I mean there's no `private $listenerProvider` in the class itself.

Reply:
Yes, but I need it to test the events.